src/style.css

```
@@ -1,3 +1,80 @@
+html {
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+h1 {
+  text-align: center;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Medium.ttf);
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font:
+    12px Roboto,
+    Arial,
+    sans-serif;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+nav {
+  margin-right: 50px;
+}
```

Remove it and add padding to the header
```suggestion
}
```